Makefile

```diff
@@ -310,6 +311,20 @@ endif
 			fi \
 		done \
 	done
+else
+	# libjulia in Darwin framework has special location and name
+	$(INSTALL_M) $(build_libdir)/libjulia.$(SOMAJOR).$(SOMINOR).dylib $(DESTDIR)$(prefix)/$(framework_dylib)
+	@$(DSYMUTIL) -o $(DESTDIR)$(prefix)/$(framework_resources)/$(FRAMEWORK_NAME).dSYM $(DESTDIR)$(prefix)/$(framework_dylib)
+	@$(DSYMUTIL) -o $(DESTDIR)$(prefix)/$(framework_resources)/sys.dylib.dSYM $(build_private_libdir)/sys.dylib
+ifeq ($(BUNDLE_DEBUG_LIBS),1)
+	# Codesign fails if Julia_debug is at root of framework next to Julia dylib.
```

Comment:
Do you have any idea why `codesign` fails?  Can you post the error here?

Reply:
I think codesign identifies a framework bundle structure and thus restricts the directory structure per the strict bundle structure style for codesigning.  I found that only one executable/library at the top must be named like `Name.framework/Versions/A/Name` and everything else under `Name.framework/Versions/A` must be symlinks and directories.  Thus `Name_debug` is rejected by codesigning.  Here's the error:
```
codesign -s "-" -v /Users/*/Documents/julia.noindex/buildmaster/julia-72c6281832/Julia.framework/Versions/1.2
/Users/*/Documents/julia.noindex/buildmaster/julia-72c6281832/Julia.framework/Versions/1.2: code object is not signed at all
In subcomponent: /Users/*/Documents/julia.noindex/buildmaster/julia-72c6281832/Julia.framework/Versions/1.2/Julia_debug
```

Update: I thought I tried this, but it turns out if `Julia_debug` is separately and explicitly signed before signing the whole framework, then `Julia_debug` doesn't need to be moved and symlinked.  Update pushed.